Makefile

```
@@ -64,7 +64,18 @@ react-app-test: | $(REACT_APP_NODE_MODULES_PATH) react-app-lint
 	cd $(REACT_APP_PATH) && yarn test --no-watch --coverage
 
 .PHONY: test
+test:
+
+# If we only want to only test go code we have to change the test target
+# which is called by all.
+ifeq ($(MAKECMDGOALS),go-only)
+test: common-test
+else
 test: common-test react-app-test
+endif
+
+.PHONY: go-only
+go-only: all
```

```suggestion
ifeq ($(GO_ONLY),)
test: common-test react-app-test
else
test: common-test
endif
```

and then run make GO_ONLY=1

that seems cleaner to me

---

wait.. actually we do not need this at all

---

Agreed